test/quantization/test_quantized_op.py

```diff
@@ -2734,6 +2734,109 @@ def test_qrnncell(self, num_batches, input_size, hidden_size, per_channel_quant)
                 self.assertEqual(result_ref[0], result_dynamic[0], msg="torch.quantized_rnncell results are off")
 
 
+"""Tests recurrent layers."""
+class TestPTQRNN(TestCase):
```

Comment:
can you merge this test case with TestQuantizedOps: https://github.com/pytorch/pytorch/blob/master/test/quantization/test_quantized_op.py#L124